tests/defaults.yml

```diff
@@ -1 +1,7 @@
 # Parameters
+parameters:
+  openshift4_nodes:
+    kubeletConfigs:
+      workers:
+        kubeletConfig:
+          maxPods: 999
```

Comment:
The defaults are meant to test, well, the defaults. With that override, We will lose that. It is possible, however, to add a second test case. Please do so.